e2e-tests/src/staking.rs

```diff
@@ -92,8 +95,110 @@ pub fn wait_for_era_completion(
 ) -> anyhow::Result<BlockNumber> {
     let sessions_per_era: u32 = connection
         .get_constant("Staking", "SessionsPerEra")
-        .unwrap();
+        .expect("Failed to decode SessionsPerEra extrinsic!");
     let first_session_in_next_era = next_era_index * sessions_per_era;
     wait_for_session(connection, first_session_in_next_era)?;
     Ok(next_era_index)
 }
+
+pub fn check_non_zero_payouts_for_era(
+    node: &String,
+    stash: &KeyPair,
+    connection: &Connection,
+    era: BlockNumber,
+) {
+    let stash_account = AccountId::from(stash.public());
+    let locked_stash_balance_before_payout = locks(&connection, &stash).expect(
+        &format!(
+            "Expected non-empty locked balances for account {}!",
+            stash_account
+        )[..],
+    );
+    assert_eq!(
+        locked_stash_balance_before_payout.len(),
+        1,
+        "Expected locked balances for account {} to have exactly one entry!",
+        stash_account
+    );
+    payout_stakers(node, stash.clone(), era - 1);
+    let locked_stash_balance_after_payout = locks(&connection, &stash).expect(
+        &format!(
+            "Expected non-empty locked balances for account {}!",
+            stash_account
+        )[..],
```

Comment:
you do not need to use `[..]` (here and in two other places)

Reply:
You're right, removed. 
Also I resolved conflicts from `main`.